removed debug menu entries

thalbern opened the pull request:
removed debug menu entries from GameObject top menu (MRTK > Renderer Size , MRTK > Collider Size)
fixes https://github.com/microsoft/MixedRealityToolkit-Unity/issues/7864

thalbern commented:
/azp run

RogPodge commented:
Echoing Kurtis's sentiments in the linked issue:
=====
> After looking further into this I think this is just some debug functionality that sneaked in there by accident
> 

From what I remember, these were added intentionally in #7584:

> This PR adds a editor utility that allows one to get the Renderer or Collider world space bounds sizes for a selected GameObject.
> Right click gameobject > MRTK > Render/Collider Size => See output results in debug log console
>

I do agree that its placement in the menu is confusing though. Does it make more sense if it's pushed down to the other "action" (non-create) items in that menu? "Move To View" etc
=======
I think changing the position of these options as well as renaming it to be more representative of their functionality is better, though these options don't seem to do much more than emit some non-descriptive debug statements @keveleigh

thalbern commented:
I've added the following changes:

- moved entries to bottom of create menu
- renamed MRTK submenu to MRTK Debug Utilities
- added word Print to the function calls in the menu so it's clear to the user that this is logging into the console
- added validate functions to have the options greyed out in the top menu in case gameobjects are selected that don't have a collider or renderer
- added missing check for empty collider list on querying collider size

However on having a closer look I'm not sure what the intention of the collider method is? atm this will only printing the collider bounds size of the first selected gameobject (that gameobject has to have the collider attached - selecting a parent without collider doesn't work) - Is this intended? I feel like these methods (probably not only collider but also renderer) should calculate:
- the combined bounds of all selected gameobjects (renderer in one case, collider in the other)
- optionally you might want this to go through the children and do a calculation on the combined bounds of all children in the hierarchy?

keveleigh commented on Assets/MRTK/Core/Utilities/Editor/SizeUtilities.cs:
```diff
@@ -34,10 +34,22 @@ public static void RendererSize()
             }
         }
 
+        [MenuItem("GameObject/MRTK Debug Utilities/Print Renderer Size", true, 41)]
```
It looks like this `MenuItem` has the same name as line 17 (same with the other two `MenuItem`s). Should this one be something like 
```suggestion
        [MenuItem("GameObject/MRTK Debug Utilities/Validate Renderer", true, 41)]
```
though, it also doesn't look like this prints anything?

thalbern replied:
it's intentional that it has the same menu item - the validation method will just make sure that menu item is grayed out - (you flag a validation method by the bool in the menuitem attribute)

thalbern replied:
ahhh but now i just realize that this one has the wrong priority - which didn't seem to bother unity for some reason - but i'll patch it up anyways :D , should be 40, not 41

keveleigh replied:
Oh cool! I didn't realize that was a thing, awesome!

thalbern commented:
/azp run
